lib/platform/common.ts

```diff
@@ -175,7 +170,10 @@ export interface Platform {
   getRepoForceRebase(): Promise<boolean>;
   deleteLabel(number: number, label: string): Promise<void>;
   setBranchStatus(branchStatusConfig: BranchStatusConfig): Promise<void>;
-  getBranchStatusCheck(branchName: string, context: string): Promise<string>;
+  getBranchStatusCheck(
+    branchName: string,
+    context: string
+  ): Promise<BranchStatus | null>;
```

Comment:
What state is `null`? Shouldn't we return `green` instead? Otherwise we have to check for null and state.

Reply:
Not sure "status not found" should equal green though - maybe yellow